common/src/main/java/com/redhat/cloud/notifications/models/Endpoint.java

```diff
@@ -68,49 +55,31 @@ public class Endpoint extends CreationUpdateTimestamped {
     @Enumerated(EnumType.STRING)
     private EndpointStatus status = EndpointStatus.UNKNOWN;
 
-    @Valid
-    @NotNull
     @Embedded
-    @JsonIgnore
+    @NotNull
+    @Valid
     private final CompositeEndpointType compositeType = new CompositeEndpointType();
```

Comment:
There's a Jackson annotation in `CompositeEndpointType` which needs to be removed.

Reply:
Removed!